### PR TITLE
PD-347   SCALE CLI - Create Network IPMI Article

### PR DESCRIPTION
**Pull request by tonyriv3**

### CODE REVIEW

File: content/SCALECLIReference/Network/CLIIPMI.md [hunk 1]
```diff
@@ -10,8 +10,158 @@ tags:
...
+
+The `channels` command returns a list of available IPMI channels.
+
+{{< expand "Viewing Available Channels" "v" >}}
+
+#### Description
+
+The `channels` command does not require entering properties or arguments.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a number showing how many IPMI channels the system has available.
```
Comment (DjP-iX) [inline content/SCALECLIReference/Network/CLIIPMI.md:36]:
```suggestion
The command returns the number of IPMI channels the system has available.
```

File: content/SCALECLIReference/Network/CLIIPMI.md [hunk 2]
```diff
@@ -10,8 +10,158 @@ tags:
...
+
+### Update Command
+
+The `update` command allows you to update the settings for a specified IPMI instance.
+
+{{< expand "Updating IPMI Settings" "v" >}}
+
+#### Description
+The `update` command has six configuration properties. They are ipaddress, netmask, gateway, password, dhcp, and vlan.
+You must enter a channel and at least one of the properties for the command to succeed.
```
Comment (DjP-iX) [inline content/SCALECLIReference/Network/CLIIPMI.md:130]:
```suggestion
You must enter a channel and at least one property for the command to succeed.
```

File: content/SCALECLIReference/Network/CLIIPMI.md [hunk 3]
```diff
@@ -10,8 +10,158 @@ tags:
...
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns nothing when successful.
```
Comment (DjP-iX) [inline content/SCALECLIReference/Network/CLIIPMI.md:132]: I believe the text we've standardized on for this type of command reply is "The command returns a blank line."

**Review by DjP-iX — CHANGES_REQUESTED** [review]

File: content/SCALECLIReference/Network/CLIIPMI.md [hunk 4]
```diff
@@ -10,8 +10,158 @@ tags:
...
+{{< /expand >}}
+
+### Update Command
+
+The `update` command allows you to update the settings for a specified IPMI instance.
+
+{{< expand "Updating IPMI Settings" "v" >}}
+
+#### Description
+The `update` command has six configuration properties. They are ipaddress, netmask, gateway, password, dhcp, and vlan.
```
Comment (micjohnson777) [inline content/SCALECLIReference/Network/CLIIPMI.md:129]: are these required or optional? Are they shared with the create command? these properties listed should be in the command emphasis. You can put a reference to the table as well.
```suggestion
The `update` command has six configuration properties. They are `ipaddress`, `netmask`, `gateway`, `password`, `dhcp`, and `vlan` explained in detail in **Update Properties** below.
```

File: content/SCALECLIReference/Network/CLIIPMI.md [hunk 5]
```diff
@@ -10,8 +10,158 @@ tags:
...
+You must enter a channel and at least one of the properties for the command to succeed.
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns nothing when successful.
+
+{{< expand "Update Properties" "v" >}}
+
+{{< truetable >}}
+| Property | Description | Syntax Example |
+|-------------|-------------|-------------|
+| `ipaddress` | IPv4 address to assign to the channel. | <code>ipaddress=<i>"ipaddress"</i></code> |
```
Comment (micjohnson777) [inline content/SCALECLIReference/Network/CLIIPMI.md:139]: the double quotes are required in the syntax so move outside the italics tags. 
```suggestion
| `ipaddress` | IPv4 address to assign to the channel. | <code>ipaddress="<i>ipaddress</i>"</code> |
```

File: content/SCALECLIReference/Network/CLIIPMI.md [hunk 6]
```diff
@@ -10,8 +10,158 @@ tags:
...
+| `netmask` | Subnet mask associated with the IP address. | <code>netmask=<i>"expandednetmask"</i></code> |
+| `gateway` | IPv4 address used by the `ipaddress` to reach outside the local subnet. | <code>gateway=<i>"gateway"</i></code> |
```
Comment (micjohnson777) [inline content/SCALECLIReference/Network/CLIIPMI.md:141]:
```suggestion
| `gateway` | IPv4 address used by the `ipaddress` to reach outside the local subnet. | <code>gateway="<i>gateway</i>"</code> |
```

File: content/SCALECLIReference/Network/CLIIPMI.md [hunk 7]
```diff
@@ -10,8 +10,158 @@ tags:
...
+| `password` | Password to assign to the channel. | <code>password=<i>password</i></code> |
```
Comment (micjohnson777) [inline content/SCALECLIReference/Network/CLIIPMI.md:142]: does this one require double quotes? what if they include a special character in the password?

**Review by micjohnson777 — DISMISSED** [review]
just a few things with emphasis, and a few questions.

File: content/SCALECLIReference/Network/CLIIPMI.md (hunk 7 above)
Comment (tonyriv3) [reply content/SCALECLIReference/Network/CLIIPMI.md:142]: No quotes for this one. I'll enter the rules for it.

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**